python/rapidsmpf/rapidsmpf/tests/conftest.py

```diff
@@ -22,9 +22,34 @@
     from rapidsmpf.communicator.communicator import Communicator
 
 
-def _get_mpi_module_or_skip() -> MPI:
+def pytest_addoption(parser: pytest.Parser) -> None:
+    """Add custom command line options for pytest."""
+    parser.addoption(
+        "--disable-mpi",
+        action="store_true",
+        default=False,
+        help="Disable MPI tests",
+    )
+
+
+@pytest.fixture(scope="session")
+def _mpi_disabled(pytestconfig: pytest.Config) -> Any:
```

Comment:
I guess `pytestconfig.getoption("--disable-mpi")` return a boolean?

```suggestion
def _mpi_disabled(pytestconfig: pytest.Config) -> bool:
```

Reply:
Unfortunately this alone fails mypy:

```
python/rapidsmpf/rapidsmpf/tests/conftest.py:40: error: Returning Any from function declared to return "bool"  [no-any-return]
Found 1 error in 1 file (checked 87 source files)
```

I agree with you this should return a `bool`, but I think because this is a `pytestconfig` it's not properly evaluated during mypy and possibly `None` is seen as a return value. What we can do is cast this explicitly to `bool` which is valid so I'll do that too.